build-helm-chart.sh

```diff
@@ -19,6 +19,19 @@ SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
 
 VERSION=$(cat $SCRIPT_DIR/VERSION)
 
+if ! command -v helm &> /dev/null
```

Comment:
This will silence any output, which could make debugging difficult; at the same time anything written to stderr will get printed by the console -- which could be confusing.

Does Helm provide unique error codes for different error cases? If so, you're we're losing that information here.

Lastly, why are you assuming that Helm cannot be found is the only reason an error condition could be hit here?

Shouldn't the existence of Helm be determined by `which helm` first?

Reply:
This command also checks the existence of helm similarly to `which helm`.

Reply:
ok